p4-16/spec/P4-16-spec.mdk

```diff
@@ -6680,6 +6752,7 @@ e.g. `bit<32> x = rand.get();`.
 | control     |  no       | yes     | no        | no      | no     |
 | extern      | yes       | yes     | yes       | yes     | no     |
 | table       |  no       | yes     | no        | no      | no     |
+| value-set   |  no       | no      | no        | no      | no     |
```

Comment:
I think it might be reasonable to put a "yes" in the "parser state" column for the value-set row, if you also say above after the sentence:

"Calling a parser, control, or table means invoking its `apply()` method."

something like this:

"Calling a value-set means using it in a `select` expression."